packages/wp-source/src/__tests__/__snapshots__/actions.tests.ts.snap

```diff
@@ -1,23 +1,22 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`fetch does nothing if data exists 1`] = `
+exports[`fetch Throw an error if fetch fails 1`] = `
 Object {
   "/some/route/": Object {
-    "id": 1,
-    "isFetching": false,
-    "isPostType": true,
-    "isReady": true,
-    "type": "example",
+    "isFetching": true,
+    "isReady": false,
   },
 }
 `;
 
-exports[`fetch returns 404 is fetch fails 1`] = `
+exports[`fetch does nothing if data exists 1`] = `
```

Comment:
This is actually a new test, but in the diff it looks like the old test was modified unfortunately.